pages/GPS-Marker-Tool.md

```diff
@@ -0,0 +1,10 @@
+GPS Marker Tool is a tool in [GPS](https://github.com/Slimefun/Slimefun4/wiki/GPS) category.
+
+## Obtaining
+GPS Marker Tool can be crafted in an [Enhanced Crafting Table](https://github.com/Slimefun/Slimefun4/wiki/Enhanced-Crafting-Table).
+
+## Usage
+GPS Marker Tool is used to create waypoints used by a [Teleporter](https://github.com/Slimefun/Slimefun4/wiki/Teleporter).
+
+Right-clicking with the GPS Marker Tool prompts user to write waypoint into chat. The waypoint is then added with given name.
```

Comment:
sentence could really be improved here
`prompts the user to write a name for their waypoint in the chat.`